src/components/Header/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+type Props = {
+  newTodoTitle: string,
+  onTitleChange: (arg: string) => void,
+  onFormSubmit: (e: React.FormEvent) => void,
+};
+
+export const Header: React.FC<Props> = ({
+  newTodoTitle,
+  onTitleChange,
+  onFormSubmit,
+}) => (
+  <header className="header">
+    <form onSubmit={onFormSubmit}>
+      <input
+        // eslint-disable-next-line jsx-a11y/no-autofocus
+        autoFocus
```

Comment:
Avoid using autoFocus on input elements to improve accessibility and user experience. Consider using a click event or other user interactions to focus on the input element instead.